lib/tasks/console.rake

```diff
@@ -1,4 +1,4 @@
-task :console => :environment do
+task :console => [:environment, 'dynflow:client'] do
```

Comment:
Does dynflow:client require `:environment` and is listing `dynflow:client` sufficient?

Reply:
Currently it doesn't, but maybe it should :thinking:

Reply:
I think it should IMHO
Could we do it as part of this PR @adamruzicka ?

Reply:
Done